T/t8code/build_tarballs.jl

```diff
@@ -11,14 +11,15 @@ version = v"1.1.2"
 # Collection of sources required to complete build
 sources = [
     ArchiveSource("https://github.com/DLR-AMR/t8code/releases/download/v$(version)/t8code_v$(version).tar.gz",
-                  "0bd4bee6694735d14fb4274275fb8c4bdeacdbd29b257220c308be63e98be8f7"),
+                  "8a30206a8fb47013b3dafe7565cf8e09023df8373c1049e7e231d9fd36b011e4"),
```

Comment:
Uhm, the tag is now pointing to a different commit?

Reply:
Yes, I updated the t8code tarball. This points to a pre-release version of t8code which will be released in the near to midterm future. I cannot wait for that and need an official jll package as soon as possible.